tox.ini

```diff
@@ -59,7 +59,7 @@ deps =
     setuptools==59.6.0
     protobuf>=3.20,<=3.20.1
     git+https://github.com/LedgerHQ/apduboy.git@d167f620430afeed313edc9fc1315d6d9c50234e
-    git+https://github.com/kevinheavey/anchorpy.git@a3cc292574679bae1610e01ab69161b6614bca92
```

Comment:
conflicts with tomte[tests] and we can it remove for now.
tests works well without this global dependency

Reply:
Perfect, tets pass, and tomte is defined in tests yeah, if so i reckon this LGTM.
I will merge into my branch then, when i am spending my free time I will go ahead and confirm the validity of this pr, and pull dassys changes in from my fork.